README.md

```diff
@@ -176,6 +176,17 @@ latency: 99.9% < 10ms
 
 ![image](https://raw.githubusercontent.com/apache/incubator-kvrocks/master/docs/images/chart-threads.png)
 
+
+## Contact Us
+
+If you’d like to contribute to Kvrocks or contact us, please subscribe to the Kvrocks developer mailing list.
+
+The Kvrocks developer mailing list is: [dev@kvrocks.apache.org](mailto:dev@kvrocks.apache.org).
+
+[Subscribe to List](dev-subscribe@kvrocks.apache.org)  
+[Unsubscribe from List](dev-unsubscribe@kvrocks.apache.org)  
+[Archives](http://mail-archives.apache.org/mod_mbox/kvrocks-dev/)
```

Comment:
```suggestion
If you'd like to contribute to Kvrocks or contact us, please [subscribe](dev-subscribe@kvrocks.apache.org) to the Kvrocks [developer mailing list](mailto:dev@kvrocks.apache.org).

You can also read archived threads at http://mail-archives.apache.org/mod_mbox/kvrocks-dev/.
```

Reply:
@tisonkun Thanks for your comment. But no unsubscribe enter when following the comment. It is not very friendly and general for end user?

Reply:
It's a "contact us" section, though. We can talk in details about playing with mailing list in the website.

Reply:
Agree. Sorry for the late response. I mean that we should offer the access and exit way before website is ready rather than only access way only. Since I found that someone already try to subscribe maillist to dev@kvrocks.apache.org directly. Of course, we have chance to update here after website is ready.

Reply:
@Hexiaoqiao It's up to you. I don't have a strong feeling on this point.